index.html

```diff
@@ -207,163 +223,65 @@ <h3>Languages, Language Tags and Matching of Language Tags</h3>
+           <p class="advisement">Best practices appear with a different background color and decoration like this.</p>
+        </aside>
+          
+        <section id="defining">
+	    <h3>How to Define Language Tags and How to Reference Language Tag Standards</h3>
+        <p class="advisement">Specifications for the Web that require language identification MUST refer to [[BCP47]]. </p>
+        
+        <p class="advisement">Specifications SHOULD NOT refer to specific component RFCs.</p>
+        <p>The "BCP" nomenclature refers to the current set of RFCs that form the "best current practice". At the time this document was published, [[BCP47]] consisted of two RFCs: <cite>Tags for the Identification of Languages </cite>[[RFC5646]] and <cite>Matching of Language Tags</cite> [[RFC4647]].</p>
+        
+        <p class="advisement">Formulations such as "<span class="quote">RFC 5646 or its successor</span>" MAY be used, but only in cases where the specific document version is necessary.</p>
+        <p>While this style of reference was once popular, using the
           BCP reference is more accurate. Since the grammar of language tags has
           been fixed since [[RFC4646]], referring to the BCP will not incur
           additional compliance risk to most implementations.</p>
-        <p>Some specifications might desire or require compliance with the older
-          language tag grammar, such as found in [[RFC1766]] or [[RFC3066]].
-          This grammar was more permissive and is described in [[!BCP47]] as the
-          production <code>obs-language-tag</code>. Specifications MUST
-          reference [[!BCP47]] and, when backwards compatibility is required,
-          SHOULD reference <code>obs-language-tag</code> rather than
-          referencing one of the obsolete versions of [[!BCP47]]. The obsolete
-          versions referred to by <code>obs-language-tag</code> include
-          [[RFC1766]] and [[RFC3066]]. [[RFC4646]], which introduced the current
-          grammar for language tags, is also obsolete.</p>
-        <p>Specifications MAY also reference registered extensions to [[!BCP47]]
-          as necessary. In particular, [[!RFC6067]] defines the <em>BCP 47
-            Extension U</em>, also known as "Unicode Locales". This extension to
-          [[!BCP47]] provides additional subtag sequences for selecting specific
-          locale variations.</p>
-        <div class="issue">
-          <p><em>Ed.:</em> Add discussion of when to use obs-language-tag. Add
-            discussion and requirements related to the terms 'well-formed' and
-            'valid'.</p>
-        </div>
+          
+        <p class="advisement">Specifications MUST NOT reference obsolete versions of [[BCP47]], such as [[RFC1766]] or [[RFC3066]].</p>
```

Comment:
General comment: the "MUST, "MUST NOT" and "SHOULD NOT" statements do not have IDs. Should these be added, to make referencing such statements later easier?

Reply:
Definitely. I added IDs.